src/test/java/com/openlattice/rehearsal/SetupTestData.kt

```diff
@@ -68,10 +64,36 @@ open class SetupTestData : MultipleAuthenticatedUsersBase() {
                     "-${ShuttleCliOptions.CREATE}=$email"))
         }
 
+        /**
+         * Import datasets via Shuttle
+         * @param
+         */
+        fun importAtlasDataSet(flightFileName: String, flightConfigurationName: String, flightConfigurationSource: String) {
```

Comment:
Is this going to be used in rehearsal tests or is it testing shuttle funcionalities? If later, I would consider moving the test there

Reply:
The goal was to create a test integration with thousands of records, hopefully allowing us to repeat the issues with linker locally.  Not really a new test, just a larger scale for testing. I think this would be the best place?

Reply:
If you build testcases in rehearsal based on the imported data, I agree, but if this is only used to import data and then manually or elsewhere test something on it, I would move it there or just do a manual import.